theano/gof/cmodule.py

```diff
@@ -2289,14 +2289,14 @@ def print_command_line_error():
         status = p_out[2]
 
         if status:
-            print('===============================')
-            for i, l in enumerate(src_code.split('\n')):
-                # gcc put its messages to stderr, so we add ours now
-                print('%05i\t%s' % (i + 1, l), file=sys.stderr)
-            print('===============================')
+            tf = tempfile.NamedTemporaryFile(delete=False)
```

Comment:
Can we have a prefix of the file be something like:

theano_compilation_error*